server/data/incidentReportingApi.ts

```diff
@@ -36,7 +40,7 @@ export type Paginated<T> = {
 export type PaginatedEventsWithBasicReports = Paginated<EventWithBasicReports>
 export type PaginatedBasicReports = Paginated<ReportBasic>
 
-export type EventWithBasicReports = {
+export type Event = {
```

Comment:
`Event` is another potentially bad/ambiguous name (I was almost tempted to rename to `Incident` as this is really the parent incident).

The type `Event` [is already a thing in JS](https://developer.mozilla.org/en-US/docs/Web/API/Event) and there are also events in Node.js.

If that's the name in the domain so be it (although I still think there are really "incidents") but it's something to be wary.